tests/test_config_parsing/test_analysis_config.py

```diff
@@ -1,6 +1,5 @@
 import os
 
-import pytest
```

Comment:
Sure about this one? `NameError: name 'pytest' is not defined`

Markers require it still

Reply:
Oops, this one was not going to be in there.